es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    super();
+    this.width = width;
+    this.area = width * width;
+    this.circumference = width * 4;
```

Comment:
So, we will have to "override" Shape's constructor values (area and circumference) from Square. How do we do that?